Core/include/Acts/Clusterization/Clusterization.hpp

```diff
@@ -11,8 +11,7 @@
 #include <memory>
 #include <vector>
 
-namespace Acts {
-namespace Ccl {
+namespace Acts::Ccl {
```

Comment:
not sure if this is a guideline but we only use nested namespaces in Acts as far as I could see

Reply:
I see them as equivalent, with a slight preference towards concatenating. I think we have this in the codebase here and there already.